tests/network-tests/src/fixtures/membership/VerifyValidatorAccountFixture.ts

```diff
@@ -0,0 +1,49 @@
+import { assert } from 'chai'
+import Long from 'long'
+import { SubmittableExtrinsic } from '@polkadot/api/types'
+import { ISubmittableResult } from '@polkadot/types/types/'
+import { WorkerId, ForumPostId } from '@joystream/types/primitives'
+import { RemarkMetadataAction } from '@joystream/metadata-protobuf'
+import { Api } from '../../Api'
+import { QueryNodeApi } from '../../QueryNodeApi'
+import { EventDetails } from '../../types'
+import { Utils } from '../../utils'
+import { BaseQueryNodeFixture } from '../../Fixture'
+
+export type ValidaotrAccountInput = {
+  memberId: string
+  isVerified: boolean
+  asWorker?: string
+}
+
+export class VerifyValidatorAccountFixture extends BaseQueryNodeFixture {
```

Comment:
I think this fixture should renamed too:
```suggestion
export class VerifyValidatorProfileFixture extends BaseQueryNodeFixture {
```